src/Sylius/Bundle/OrderBundle/Controller/OrderItemController.php

```diff
@@ -32,6 +32,7 @@
 use Symfony\Component\HttpKernel\Exception\HttpException;
 use Symfony\Component\Validator\ConstraintViolationListInterface;
 
+/** @deprecated since Sylius 1.14 and will be removed in Sylius 2.0. */
```

Comment:
lets make it align with the other deprecations, like [here](https://github.com/Sylius/Sylius/blob/1.14/src/Sylius/Bundle/AdminBundle/Controller/Dashboard/StatisticsController.php#L21-L27)